tailwind.config.js

```diff
@@ -2,7 +2,34 @@
 module.exports = {
     content: ["./pages/**/*.{js,ts,jsx,tsx}", "./src/**/*.{js,ts,jsx,tsx}"],
     darkMode: "media",
-    theme: {},
+    theme: {
+        extend: {
+            colors: {
+                primary: {
+                    100: "#cffafe",
+                    200: "#a5f3fc",
+                    300: "#67e8f9",
+                    400: "#22d3ee",
+                    500: "#06b6d4",
+                    600: "#0891b2",
+                    700: "#0e7490",
+                    800: "#155e75",
+                    900: "#164e63"
+                },
+                secondary: {
+                    100: "#f2e8e5",
+                    200: "#eaddd7",
+                    300: "#e0cec7",
+                    400: "#d2bab0",
+                    500: "#bfa094",
+                    600: "#a18072",
+                    700: "#977669",
+                    800: "#846358",
+                    900: "#43302b"
+                }
+            }
+        }
+    },
```

Comment:
@dimitribarbot 

Thank you for this PR.
We like the idea of having `primary` and `secondary` colors for the component.

`theme` configs in the `tailwind.config.js` file can only be used during development while using the [Playground](https://github.com/onesine/react-tailwindcss-datepicker/blob/master/README.md#playground). 
Thus, adding custom tailwind config may not work in production.